blog/_posts/2022-09-29-scala-survey-2022-announcement.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog-detail
+post-type: blog
+by: Kacper Korban, VirtusLab
+title: Participate in the Scala Survey 2022!
+---
+
+![Scala Survey 2022 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2022-announcement.png)
+
+## The Scala Team wants to hear from you!
+
+We want to make the language we all love as amazing as possible. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why ask you to help by participating in the Scala Survey 2022.
```

Comment:
"that's why _we_ want to..."

Maybe consider a more open style "we invite you to..."